src/components/header/index.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import TodoTextInput from '../../containers/todo_text_input/container';
+
+type HeaderProps = {
+  addTodo?: () => any;
+}
+
+const placeholder = 'What needs to be done?';
+
+export default ({ addTodo }: HeaderProps) => (
+  <header className='header'>
+    <h1>todos</h1>
+    <TodoTextInput isNew placeholder={placeholder} onSubmit={addTodo} />
```

Comment:
Is the `isNew` bit valid?

Reply:
the `isNew` is a class to make the top one look different